### DISCUSSION TITLE
Sending keystrokes to Raycast

### DISCUSSION THREAD

**Discussion by igorterroso** [ttscoff/bunch-docs#251] [Q&A] [answered]
Hi guys,

I am new to Bunch but I am quickly becoming totally addicted.

I am trying to send keystrokes to control window placement using Raycast but having no success. Has anyone had this problem?

Command being sent is:
```

Todoist
- {control-command-right}

```
control-command-right is the shortcut I have set in Raycast to set the active window to a certain position.

Any help would be appreciated :-D

**Comment by ttscoff** [comment, marked as answer]
I believe you're trying to send a global shortcut, in which case you should put the keystroke command on its own line and not as a file argument for todoist.

Thanks,
Brett

**Reply by igorterroso** [reply]
Thanks for the answer! It makes sense but now I have another issue.

How do I ensure the app I need to resize is loaded and focused before sending the keystrokes? Currently it sends them right when I trigger the bunch although I have sequential execution activated in the frontmatter.

**Comment by ttscoff** [comment]
Use a Waiting Snippet, optionally with ~ delay.

https://bunchapp.co/docs/bunch-files/snippets/#waitingsnippet

Thanks,
Brett
